tasks/human_eval_031.rs

```diff
@@ -10,40 +10,26 @@ use vstd::prelude::*;
 verus! {
 
 // specification
-pub closed spec fn prime_helper(n: nat, k: nat) -> bool
+pub closed spec fn is_prime_so_far(n: nat, k: nat) -> bool
```

Comment:
This is very minor, but I think the style in the existing tasks is to put the trusted specification (e.g., `is_prime`) at the top of the file, while helper functions and lemmas are lower down.  I would tend to think of and label those as proof, even if they define/use `spec` functions, since they're not trusted specifications, they exist only to simplify the proof process.

Reply:
good to know